include/private/vkd3d_common.h

```diff
@@ -250,4 +250,22 @@ static inline uint32_t float_bits_to_uint32(float f)
     return u;
 }
 
+static inline size_t vkd3d_wcslen(const WCHAR *wstr, size_t wchar_size)
+{
+    const char *data = (const char*)wstr;
+    uint32_t curr_char;
+    size_t length = 0;
+
+    while (true)
+    {
+        memset(&curr_char, 0, sizeof(curr_char));
```

Comment:
Why memset a uint32_t?

Reply:
Seems easier to have a uint16_t variant and a uint32_t variant than dealing with memset/memcmp.

Reply:
Fixed.